packages/simm-validation/src/is-mobile-phone.ts

```diff
@@ -0,0 +1,46 @@
+import { regexValidate } from "./shared/constants/regex";
+
+type MobilePhoneLocale = keyof typeof regexValidate.MOBILE_PHONE_NUMBER;
+
+/**
+ * Validates if the given string is phone number or not.
+ *
+ * @param {any} input The string to validate.
+ * @param {MobilePhoneLocale} locale The locale of phone.
+ * @returns {boolean} Returns true if the string is phone, false otherwise.
+ */
+const isMobilePhone = (
+  input: any,
+  locale?: MobilePhoneLocale | MobilePhoneLocale[]
+) => {
+  // Returns false input is not a string
+  if (typeof input !== "string") return false;
+
```

Comment:
![image](https://github.com/user-attachments/assets/50964a4b-9387-410e-843d-fbe268b8cfb8)
logic 3 chỗ này giống nhau thì tách ra function checkLocale đi e
```suggestion
const checkLocale = (loc: MobilePhoneLocale): boolean => {
    const phoneRegex = regexValidate.MOBILE_PHONE_NUMBER[loc];
    return phoneRegex ? phoneRegex.test(input) : false;
  };
```